app/templates/checklist/view_checklist_items.html

```diff
@@ -0,0 +1,84 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui stackable grid container">
+      <div class="ui hidden divider"></div>
+        <div class="sixteen wide tablet twelve wide computer centered column">
+            <h2 class="ui header">
+                Checklist Items
+                <div class="sub header">
+                    View and manage checklist items.
+                </div>
+            </h2>
+            <div class="ui middle aligned divided list">
+                <div class="item">
+                    <div class="right floated content">
+                        <a href="" class="ui button">
+                            <i class="upload icon"></i>
+                            Upload new document
+                        </a>
+                    </div>
+                    <div class="content">
+                        <div class="ui left icon fluid input">
+                            <i class="search icon"></i>
+                            <input id="search-checklist-items" type="text" placeholder="Search checklist item by name">
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <table class="ui searchable sortable unstackable selectable celled table" style="overflow-x: scroll;">
+                <thead>
+                    <tr>
+                        <th>Title</th>
+                        <th>Description</th>
+                        <th class="one wide"></th>
+                    </tr>
+                </thead>
+                <tbody>
+                {% for c in default_checklist_items %}
```

Comment:
We should remove `default_checklist_items` from the user specific checklist items view.

Reply:
Got it!